### PR TITLE
add short sentence on datalad via easybuild

### PR DESCRIPTION
**Pull request by jkuhl-uni**
Hey,

as mentioned in #1117, I added a small note in the cluster section of the handbook, that datalad may be installed per Easybuild.

### CODE REVIEW

**Review by adswa — APPROVED** [review]
Thank you very much!

**Comment by adswa** [comment]
The appveyor failure is unrelated, we can merge without it passing.
Please feel free to add yourself to the zenodo.json file in this repository via PR (second to last author) or leave your name, orcid (optional), and affiliation (optional) as a comment in this thread and I'll add you :)

hey @all-contributors, please add @jkuhl-uni for content

File: docs/beyond_basics/101-169-cluster.rst [hunk 1]
```diff
@@ -18,6 +18,8 @@ DataLad installation on a cluster
 
 Users of a compute cluster generally do not have administrative privileges (sudo rights) and thus cannot install software as easily as on their own, private machine.
 In order to get DataLad and its underlying tools installed, you can either `bribe (kindly ask) your system administrator <https://hsto.org/getpro/habr/post_images/02e/e3b/369/02ee3b369a0326760a160004aca631dc.jpg>`_ [#f1]_ or install everything for your own user only following the instructions in the paragraph :ref:`norootinstall` of the :ref:`installation page <install>`.
+If you opt for the first, your administrator can install Datalad version 0.18.4 via `EasyBuild <https://github.com/easybuilders>`, which is a tool for building software reprobucibly and is common on clusters that use a module system.
+The caveat this introduces, of course, is that you will need to load the module everytime you want to use datalad on your cluster.
```
Comment (adswa) [inline docs/beyond_basics/101-169-cluster.rst:22]:
```suggestion
The caveat this introduces, of course, is that you will need to load the module every time you want to use datalad on your cluster.
```

**Comment by adswa** [comment]
hey @all-contributors please add @jkuhl-uni for content

**Comment by jkuhl-uni** [comment]
Alright, thank you too!
I just added myself to the zenodo.json. I hope it is okay to do this in this PR as well.

**Pull request merged**